aerogear/data-sync-server
# Top level auth config for server

## Motivation

Branch that is going to be used for collaboration for 

NOTES: Pushed helper files that should not land into master only to help and point out preconfigured application cluster. They going to be removed after

## Verification steps

Verification will focus on memeolist example.
We will need to run memeolist without keycloak 

1. `npm run dev:memeo` >  
2. Log should appear: Keycloak authentication is not configured
3. UI queries should work

Memeolist with keycloak.
1. `npm run dev:keymemeo` >  Initializing Keycloak authentication
2. UI queries should *not* work - keycloak login page should appear (if you provide redirect url :)
